Doc/library/os.rst

```diff
@@ -3781,6 +3781,425 @@ features:
+                  if data:
+                      # You should catch UnicodeDecodeError exception for safety.
+                      cmd = data.decode()
+                      if cmd.startswith("stop"):
+                          print(f"Stopping server")
+                          isActive = False
+                      else:
+                          print(f"Unknown command: {cmd}")
+                  else:
+                      # No more data, close connection
+                      print(f"Closing connection {fd}")
+                      select_fds.remove(conn)
+                      conn.close()
+                      conn = None
+              elif fd in fds:
+                  print(f"Reading timer {fd}")
+                  count = int.from_bytes(os.read(fd, 8), byteorder=sys.byteorder)
+                  print(f"Timer {fds.index(fd) + 1} expired {count} times")
+              else:
+                  print(f"Unknown file descriptor {fd}")
+   finally:
+       for fd in fds:
+          os.close(fd)
+       sock.close()
+       sock = None
+
+--------------
+
+.. function:: timerfd_create(clockid, /, *, flags=0)
+
+   Create and return a timer file descriptor (*timerfd*).
+
+   The file descriptor returned by :func:`timerfd_create` supports:
+
+      - :func:`read`
+      - :func:`~select.select`
+      - :func:`~select.poll`.
+
+   The file descriptor's :func:`read` method can be called with a buffer size of 8.
+   If the timer has already expired one or more times, :func:`read` returns
+   the number of expirations with the host's endianness,
+   which may be converted to an :class:`int` by ``int.from_bytes(x, byteorder=sys.byteorder)``.
+
+   :func:`~select.select` and :func:`~select.poll` can be used to wait until
+   timer expires and the file descriptor is readable.
+
+   *clockid* must be a valid :ref:`clock ID <time-clock-id-constants>`,
+   as defined in the :py:mod:`time` module:
+
+   - :const:`time.CLOCK_REALTIME`
+   - :const:`time.CLOCK_MONOTONIC`
+   - :const:`time.CLOCK_BOOTTIME` (Since Linux 3.15)
+
+   If *clockid* is :const:`time.CLOCK_REALTIME`, a settable system-wide real-time clock
+   is used. If system clock is changed, timer setting need to be updated.
+   To cancel timer when system clock is changed, see :const:`TFD_TIMER_CANCEL_ON_SET`.
+
+   If *clockid* is :const:`time.CLOCK_MONOTONIC`, A nonsettable monotonically increasing
+   clock is used. Even if system clock is changed, timer setting will be not affected.
+
+   If *clockid* is :const:`time.CLOCK_BOOTTIME`, same as :const:`time.CLOCK_MONOTONIC`
+   except it includes any time that the system is suspended.
+
+   The file descriptor's behaviour can be modified by specifying a *flags* value.
+   Any of the following variables may used, combined using bitwise OR
+   (the ``|`` operator):
+
+   - :const:`TFD_NONBLOCK`
+   - :const:`TFD_CLOEXEC`
+
+   If :const:`TFD_NONBLOCK` is not set as a flag, :func:`read` blocks until the timer expires.
```

Comment:
Too long lines. It makes reviewing difficult. I do not remember the exact limit for documentation, but it is not more than 80 columns.

Reply:
@serhiy-storchaka 
Fixed.